resources/public/api/conf/1.0/schema/register-nino.json

```diff
@@ -0,0 +1,411 @@
+  "required": [
+    "nino",
+    "gender",
+    "entryDate",
+    "birthDate",
+    "birthDateVerification",
+    "officeNumber",
+    "country",
+    "name",
+    "address"
+  ],
+  "objects": {
+    "name": {
+      "description": "Multi-Use Name Object",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "title": {
+          "description": "Name Title String",
+          "type": "string",
+          "enum": [
+            "NOT KNOWN",
+            "MR",
+            "MRS",
+            "MISS",
+            "MS",
+            "DR",
+            "REV"
+          ]
+        },
+        "forename": {
+          "description": "Individuals primary forename string",
+          "$ref": "#/definitions/nameElement"
+        },
+        "secondForename": {
+          "description": "Individuals secondary forename string",
+          "$ref": "#/definitions/nameElement"
+        },
+        "surname": {
+          "description": "Individuals surname string",
+          "$ref": "#/definitions/nameElement"
+        },
+        "startDate": {
+          "description": "Individuals name record start date",
+          "$ref": "#/definitions/date"
+        },
+        "endDate": {
+          "description": "Individuals name record end date",
+          "$ref": "#/definitions/date"
+        }
+      },
+      "required": [
+        "surname",
+        "startDate"
+      ]
+    },
+    "address": {
+      "description": "Multi-Use Address Object",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "addressType": {
+          "description": "Type of address provided",
+          "type": "string",
+          "enum": [
+            "RESIDENTIAL",
+            "CORRESPONDENCE"
+          ]
+        },
+        "line1": {
+          "description": "First address line",
+          "$ref": "#/definitions/addressLine"
+        },
+        "line2": {
+          "description": "Second address line",
+          "$ref": "#/definitions/addressLine"
+        },
+        "line3": {
+          "description": "Third address line",
+          "$ref": "#/definitions/addressLine"
+        },
+        "line4": {
+          "description": "Fourth address line",
+          "$ref": "#/definitions/addressLine"
+        },
+        "line5": {
+          "description": "Fifth address line",
+          "$ref": "#/definitions/addressLine"
+        },
+        "postcode": {
+          "description": "Address postcode",
+          "$ref": "#/definitions/postcode"
+        },
+        "countryCode": {
+          "description": "Address Country Code",
+          "$ref": "#/definitions/countryCode"
+        },
+        "startDate": {
+          "description": "Individuals address record start date",
+          "$ref": "#/definitions/date"
+        },
+        "endDate": {
+          "description": "Individuals address record end date",
+          "$ref": "#/definitions/date"
+        }
+      }
+    },
```

Comment:
This needs to have startDate and line1 as a required fields (according to document)